src/lib/features/export-import-toggles/export-import-api-usage.test.ts

```diff
@@ -0,0 +1,182 @@
+import { setupAppWithCustomAuth } from '../../../test/e2e/helpers/test-helper';
+import dbInit, { ITestDb } from '../../../test/e2e/helpers/database-init';
+import getLogger from '../../../test/fixtures/no-logger';
+import {
+    DEFAULT_PROJECT,
+    IContextFieldStore,
+    IEnvironmentStore,
+    IEventStore,
+    IFeatureToggleStore,
+    IProjectStore,
+    ITagStore,
+} from '../../types';
+import { DEFAULT_ENV } from '../../util';
+import { ImportTogglesSchema, VariantsSchema } from '../../openapi';
+import { ApiTokenType } from '../../types/models/api-token';
+import { ApiUser } from '../../server-impl';
+
+let db: ITestDb;
+let eventStore: IEventStore;
+let environmentStore: IEnvironmentStore;
+let contextFieldStore: IContextFieldStore;
+let projectStore: IProjectStore;
+let toggleStore: IFeatureToggleStore;
+let tagStore: ITagStore;
+
+beforeAll(async () => {
```

Comment:
do we use all of those stores from the beforeAll setup? If not we can drop them and keep the test file smaller